.github/workflows/frontend-tests.yml

```diff
@@ -5,6 +5,13 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ '*' ]
+    paths:
```

Comment:
building the production image is important in both js and python tests. Because we need it in both places, maybe we should pull it out into its own workflow file after all?

Reply:
Yeah, makes sense.